### DISCUSSION TITLE
How can I increse the size of font in tabby setting menu?

### DISCUSSION THREAD

**Discussion by kimhongsu** [Eugeny/tabby#5818] [Q&A]
Hello,
How can I increse the size of font in tabby setting menu?
The default font size in tabby setting menu is too small for me. 
I can change the terminal font size in appearance setting menu.

**Comment by germain-italic** [comment]
I faced the same issue, having to work on a 14" full HD laptop.
I tweaked the interface by adding the following line in **Settings** > **Appearance** > **Custom CSS**:

```css
app-root { font-size: 1.25em; }
```

Hope it helps!
